pyteal/ast/widemath.py

```diff
@@ -157,3 +214,45 @@ def has_return(self):
 
 
 WideRatio.__module__ = "pyteal"
+
+
+class WideUint128(MultiValue):
+    @staticmethod
+    def addw(*terms: Expr):
+        if len(terms) < 2:
+            pass
+        return WideUint128(
+            Op.addw, [TealType.uint64, TealType.uint64], args=list(terms)
+        )
+
+    @staticmethod
+    def mulw(*factors: Expr):
+        if len(factors) < 2:
+            pass
+        return WideUint128(
+            Op.addw, [TealType.uint64, TealType.uint64], args=list(factors)
```

Comment:
@ahangsu Friendly pointer to what I believe to be a typo.

```suggestion
            Op.mulw, [TealType.uint64, TealType.uint64], args=list(factors)
```